### PR TITLE
[wiki migration] Add static images for flutter/flutter/docs/

### PR DESCRIPTION
**Pull request by Piinks**
This adds all of the images that are embedded in the flutter/flutter wiki, which is being migrated to flutter/flutter/docs/.
Part of https://github.com/flutter/flutter/issues/145009

I needed to make a few other trivial updates for CI to pass, filed https://github.com/flutter/flutter/issues/148709 to follow up on the ignore I placed on ButtonBar.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/diagrams/lib/src/card.dart [hunk 1]
```diff
@@ -34,6 +34,7 @@ class CardDiagram extends StatelessWidget with DiagramMetadata {
                 subtitle: Text('Music by Julie Gable. Lyrics by Sidney Stein.'),
               ),
               ButtonBar(
+                // ignore: deprecated_member_use
```
Comment (Piinks) [inline packages/diagrams/lib/src/card.dart:37]: Filed https://github.com/flutter/flutter/issues/148709 for follow up

**Review by justinmc — APPROVED** [review]
LGTM :+1:

File: packages/diagrams/lib/src/card.dart [hunk 2]
```diff
@@ -8,6 +8,8 @@ import 'package:flutter/material.dart';
 
 import 'diagram_step.dart';
 
+// ignore_for_file: deprecated_member_use
```
Comment (goderbauer) [inline packages/diagrams/lib/src/card.dart:11]: nit: add a TODO linking to the issue you filed?

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**